proprietary/tokens/src/components/ams/description-list.tokens.json

```diff
@@ -4,15 +4,17 @@
       "color": { "value": "{ams.color.primary-black}" },
       "font-family": { "value": "{ams.text.font-family}" },
       "font-size": { "value": "{ams.text.level.5.font-size}" },
-      "font-weight": { "value": "{ams.text.font-weight.normal}" },
       "gap": { "value": "{ams.space.sm}" },
       "inverse-color": { "value": "{ams.color.primary-white}" },
       "line-height": { "value": "{ams.text.level.5.line-height}" },
       "row": {
         "gap": { "value": "{ams.space.sm}" }
       },
+      "term": {
+        "font-weight": { "value": "{ams.text.font-weight.bold}" }
+      },
       "details": {
-        "font-weight": { "value": "{ams.text.font-weight.bold}" },
+        "font-weight": { "value": "{ams.text.font-weight.normal}" },
```

Comment:
Is it necessary to set the `font-weight` if it's the default (`normal`)?

This token is not being used, correct? Or is this for legacy purposes?